.github/workflows/cron.yml

```diff
@@ -13,14 +13,18 @@ jobs:
     name: Build DB
     runs-on: ubuntu-24.04
     steps:
+      # vuln-list dirs + language repositories use more than 31GB of storage
       - name: Maximize build space
-        uses: easimon/maximize-build-space@v10
+        uses: AdityaGarg8/remove-unwanted-software@v5
```

Comment:
We should use the commit hash for security reasons.

Reply:
Right! Thanks!
Fixed in https://github.com/aquasecurity/trivy-db/pull/543/commits/43eb3a650e21c96d2ab3170901c86bfb20185394

Reply:
NOTE: I would like to avoid relying on third-party actions (especially those that are not widely used) as much as possible, so I plan to remove them in the future.

Reply:
@knqyf263 since we use this action quite a bit, should be fork it and maintain it under aquasec? That way we could eliminate security issues such as the ones you mention.

Of course if we find another way to not use such actions (which are kind of a hack anyway), it'll be the best.

Reply:
Since they are simple shell scripts, we can write a similar one from scratch.